doc/index.md

```diff
@@ -1,56 +1,59 @@
 # Getting Started
 
-Npgsql has an Entity Framework Core provider. It mostly behaves like a any other EFCore provider (e.g. SQL Server) - all the information in the [general EF Core docs](https://docs.microsoft.com/en-us/ef/core/index) applies. If you're just getting started with EF Core, those docs are the best place to start.
+Npgsql has an Entity Framework (EF) Core provider. It behaves like other EF Core provider (e.g. SQL Server), so all of the information in the [general EF Core docs](https://docs.microsoft.com/en-us/ef/core/index) applies. If you're just getting started with EF Core, those docs are the best place to start.
 
 Development happens in the [Npgsql.EntityFrameworkCore.PostgreSQL](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL) repository, all issues should be reported there.
 
-## Using the Npgsql EF Core Provider
+## Configuring the project file
 
-To use the Npgsql EF Core provider, simply add a dependency on `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
+To use the Npgsql EF Core provider, add a dependencies for `Npgsql` and `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
 
-Following is an example (new-style) csproj using Npgsql EF Core:
+Below is a `.csproj` file for a console application that uses the Npgsql EF Core provider:
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.0.0" />
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.0" />
+    <PackageReference Include="Npgsql" Version="4.0.0" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.1.0" />
   </ItemGroup>
 </Project>
 ```
 
-## Using the Npgsql provider
+## Defining a `DbContext`
 
-    public class BlogContext : DbContext
-    {
-        // When used with ASP.net core, add these lines to Startup.cs
-        //   var connectionString = Configuration.GetConnectionString("BlogContext");
-        //   services.AddEntityFrameworkNpgsql().AddDbContext<BlogContext>(options => options.UseNpgsql(connectionString));
-        // and add this to appSettings.json
-        // "ConnectionStrings": { "BlogContext": "Server=localhost;Database=blog" }
+```c#
+public class BlogContext : DbContext
```

Comment:
Looking at this again, this isn't a good sample for a console program - see [this EF Core doc page](https://docs.microsoft.com/en-us/ef/core/get-started/netcore/new-db-sqlite). This context shouldn't have a constructor, and instead should have an `OnConfiguring()` method with a `UseNpgsql()` using some hard-coded connection string - that's the simplest path to getting an application working.